lib/flappy_translator.dart

```diff
@@ -199,6 +209,14 @@ class FlappyTranslator {
     return text;
   }
 
+  String _replaceNoBreakSpaces(String text) {
+    // sometimes there can be the strange '194 160' space, ie `no break space`. replace this with a normal space.
+    text = text.replaceAll('\u00A0', ' ');
+    text = text.replaceAll(' ', ' ');
```

Comment:
Is the line 215 really necessary ?

Reply:
This line has the actual character that was the problem on the left, and a real space on the right. I didn't test too much but in some cases 214 wasn't sufficient. If you enable render whitespace on your IDE, you will see that these are two different characters.

Reply:
Ok I understand better, thank you :)